extension/src/plots/model/custom.ts

```diff
@@ -69,7 +69,16 @@ export const checkForCustomPlotOptions = (
   return false
 }
 
-export const createSpec = (title: string, metric: string, param: string) =>
+const getSpecDataType = (type: string) =>
+  type === 'number' ? 'quantitative' : 'nominal'
```

Comment:
When custom plots were first created, I had them set to `nominal` by default due to the fact that none-number values break on `quantitive`. Somewhere down the line, it must have changed back to `quantitive` 🤔.  I think it was because only `quantitive` orders the number values correctly.